test/e2e/storage/testsuites/provisioning.go

```diff
@@ -72,8 +72,8 @@ func InitProvisioningTestSuite() TestSuite {
 		tsInfo: TestSuiteInfo{
 			Name: "provisioning",
 			TestPatterns: []testpatterns.TestPattern{
-				testpatterns.DefaultFsDynamicPV,
-				testpatterns.BlockVolModeDynamicPV,
+				testpatterns.DynamicSnapshotDelete,
```

Comment:
should we remove the snapshot tests from provisioning.go if they're redundant with the ones in snapshottable.go?

Reply:
Done